templates/default.md

```diff
@@ -81,6 +81,9 @@
 <% if (authorGithubUsername) { -%>
 * Github: [@<%= authorGithubUsername %>](https://github.com/<%= authorGithubUsername %>)
 <% } -%>
+<% if (authorPatreonLink) { -%>
+* Patreon: <%= authorPatreonLink %>
```

Comment:
I was thinking of a button like the one in [this README](https://github.com/perkinsjr/authentication_boilerplate) 😉 What do you think ?

Reply:
That's nice, let's see how it looks like. But then I think it should also rename `authorPatreonLink` into `authorPatreonUsername` and ask just for the username instead of a full link.

Reply:
Yes !

Reply:
done